bin/install-chart.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+CHART="$1"
+shift # preserve "$@" for --post-renderer-args
+
+GENESTACK_DIR="${GENESTACK_DIR:-/opt/genestack}"
+CHART_META_FILE=\
+"${CHART_META_FILE:-$GENESTACK_DIR/bin/chart-install-meta.yaml}"
+GENESTACK_DIR="${GENESTACK_DIR:-/opt/genestack}"
```

Comment:
This has been mentioned twice
```suggestion
```

Reply:
@mnaghavi thanks for the review! I don't see `GENESTACK_DIR="${GENESTACK_DIR:-/opt/genestack}"` mentioned before. Why would we want to remove that?

I'm curious if you think this is meshing well with your efforts and if I should keep up along the lines of this PR or just drop it and go to individual scripts. I don't want to try pushing things in an incoherent direction.

Thanks,
Adam

Reply:
@awfabian-rs on line `6` we do have it too, just wanted to avoid repetition

of course, it keeps us avoiding using a script 6 times as you mentioned and in the future we can use this kind of scripts in the pipeline

Reply:
fixed